I do magic/pokemonFunction/damage.js

```diff
@@ -0,0 +1,89 @@
+class Pokemon {
+  constructor(name, type, attack, defense){
+    this._attack = attack;
+    this._name = name;
+    this._type = type;
+    this._defense = defense;
+  }
+  
+  get name(){
+    return this._name;
+  }
+
+  get type(){
+    return this._type;
+  }
+
+  get attack(){
+    return this._attack;
+  }
+
+  get defense(){
+    return this._defense;
+  }
+}
+
+function calculateMultiplier (typeAtk, typeDef) {
+  if(typeAtk === typeDef){
+    return 0.5;
+  }
+  else{
+    switch(typeAtk){
```

Comment:
on js try to avoid "switch" statement 

- http://www.nikola-breznjak.com/blog/javascript/converting-a-javascript-switch-statement-into-a-function-lookup/
- https://toddmotto.com/deprecating-the-switch-statement-for-object-literals/